package.json

```diff
@@ -45,6 +45,9 @@
     "vuepress-plugin-sitemap": "^2.3.1",
     "vuepress-plugin-zooming": "^1.1.8"
   },
+  "engines": {
+    "node": ">=12 <15"
```

Comment:
I think you mentioned this over slack, but why do we need the `engines` field? If I understand correctly both netlify and local give precedence to .nvmrc, no?